app/javascript/src/api/account.js

```diff
@@ -5,6 +5,78 @@
 import endPoints from './endPoints';
 
 export default {
+  /**
+   * handleGet
+   * @param dataType {string} [dataType='agent', 'label', 'inbox'] type of data requested
+   * @returns {promise}
+   */
+  handleGet(dataType) {
+    let urlData;
+    switch (dataType.toLowerCase()) {
+      case 'agent':
+        urlData = endPoints('fetchAgents');
+        break;
+      case 'label':
+        urlData = endPoints('fetchLabels');
+        break;
+      case 'inbox':
+        urlData = endPoints('fetchInboxes');
+        break;
+      default:
+        console.log('Incorrect query');
+        return;
+    }
+
+    const fetchPromise = new Promise((resolve, reject) => {
+      axios
+        .get(urlData.url)
+        .then(response => {
+          resolve(response);
+        })
+        .catch(error => {
+          reject(Error(error));
+        });
+    });
+    return fetchPromise;
+  },
+
+  /**
+   * handleDelete
+   * @param dataType {string} [dataType='agent', 'label', 'inbox'] type of data requested
+   * @param id {number} id of target data
+   * @returns {promise}
+   */
+  handleDelete(dataType, id = 0) {
+    if (!id) {
+      console.log('Incorrect query');
+      return;
+    }
+
+    let urlData
+    switch (dataType.toLowerCase()) {
```

Comment:
Move this switch case also to a function

Reply:
I also think the return in the conditional that's checking for the `id` may be making the lint tests fail.